examples/carbon-for-ibm-products/FullPageError/src/Example/Example.jsx

```diff
@@ -4,21 +4,30 @@ import React from 'react';
 
 // ----------------------------------------------------
 // This is an example showing use of the
-// AboutModal component
+// FullPageError component
 // from the @carbon/ibm-products component library.
 // ----------------------------------------------------
-
 import { FullPageError } from '@carbon/ibm-products';
-
+// import { pkg } from '@carbon/ibm-products';
+import { Link } from '@carbon/react';
 import './_example.scss';
 
+// pkg.component.FullPageError = true;
```

Comment:
```suggestion
import { Link } from '@carbon/react';
import './_example.scss';
```

Can remove these leftover comments.